crates/rpc/rpc-types/src/eth/engine/payload.rs

```diff
@@ -559,6 +569,34 @@ mod tests {
         assert_eq!(serde_json::to_string(&status).unwrap(), full);
     }
 
+    #[test]
+    fn serde_payload_status_error() {
+        let s = r#"{"status":"INVALID_BLOCK_HASH","latestValidHash":null,"validationError":
+        {"InvalidBlockHash":{"hash":"0x1241835b37d267ed64d1840312ae600b1774e48370b0548c87012a1c681aa54d"}}}"#;
```

Comment:
this is now an object, this is why I requested manual serde impls because this must be a String

Reply:
Got it. I had misunderstood the original issue. Will make this String.